manifests/init.pp

```diff
@@ -169,4 +170,9 @@
     type => 'string',
     data => $target_group,
   }
+
+  wsus_client::setting{ "${_au_base}\\AlwaysAutoRebootAtScheduledTime":
+    data           => $always_auto_reboot_at_scheduled_time,
+    validate_range => [0,1],
```

Comment:
Maybe is good idea to add AlwaysAutoRebootAtScheduledTimeMinutes to set time of force reboot after upgrade?